### ISSUE TITLE
The about page isn't responsive enough

### ISSUE THREAD

**Issue by kevinega** [Alieff/timAul#10]
The about page isn't responsive enough, the tip is to use **_bootstrap grid_** class for the avatar-images of us (team members) you can learn about **_bootstrap grid_** here http://getbootstrap.com/css/#grid.

**Comment by aulch** [comment]
Yes, thank you for the review. I already clarified about this to Puti as well. I haven't used bootstrap yet to make the images responsive as expected.
